src/main/scala/temple/generate/server/go/service/dao/GoServiceDAOFunctionsGenerator.scala

```diff
@@ -107,8 +106,8 @@ object GoServiceDAOFunctionsGenerator {
       .list(
         s"&${root.decapitalizedName}.${root.idAttribute.name.toUpperCase}",
         root.createdByAttribute match {
```

Comment:
Because we're in optional world now we can just use `.map` which automatically does the `None => None` case for you :)